DB/ReviewsController.js

```diff
@@ -7,14 +7,47 @@ const getAll = (callback) => {
 }
 
 const getById = (id, callback) => {
-  Reviews.find({ productId : `${id} ` }, (err, result) => {
+  Reviews.find({ productId : `${id}` }, (err, result) => {
     if (err) {
       callback(err, null);
     } else {
       callback(null, result);
     }
-  })
+  });
+}
+
+const deleteById = (id, callback) => {
+  Reviews.deleteOne({ productId: `${id}` }, (err, result) => {
+    if (err) {
+      callback(err, null);
+    } else {
+      callback(null, result);
+    }
+  });
+}
+
+const createReview = (review, callback) => {
+  Reviews.create(review, (err, review) => {
+    if (err) {
+      callback(err, null);
+    } else {
+      callback(null, result);
+    }
+  });
+}
+
+const updateById = (id, updateObj, callback) => {
+  Reviews.findOneAndUpdate({id : `${id}`}, updateObj, (err, result) => {
```

Comment:
I don't know that template literals are necessary here. I would think you could just use the variable without them.